@navikt/ds-react/layouts/product-page/Panel.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useState } from "react";
+import { forwardRef, HTMLAttributes, ReactNode } from "react";
+import cl from "classnames";
+import { Button, Heading } from "../../";
+import { Attachment } from "@navikt/ds-icons";
+import { EtikettInfo } from "nav-frontend-etiketter";
+import { Undertekst } from "nav-frontend-typografi";
+
+export interface PanelProps extends HTMLAttributes<HTMLDivElement> {
+  title: string;
+  icon?: ReactNode;
+  highlight?: boolean;
+  anchor?: string;
+  whiteBackground?: boolean;
+  withPadding?: boolean;
+}
+
+const Panel = forwardRef<HTMLDivElement, PanelProps>(
+  ({ icon, highlight, anchor, title, children, className, ...rest }, ref) => {
+    const [copied, setIsCopied] = useState<boolean>(false);
+    const copyRef = React.createRef<HTMLButtonElement>();
+
+    useEffect(() => {
+      const header = document.getElementById("layout-header");
+      header?.classList.add("navds-layout__header--margin");
+    }, []);
+
+    return (
+      <div
+        ref={ref}
+        className={cl(
+          "navds-layout__panel",
+          icon && "navds-layout__panel--margin",
+          highlight && "navds-layout__panel--highlight",
+          className
+        )}
+        {...rest}
+      >
+        {icon && <div className={"navds-layout__panel-icon"}>{icon}</div>}
+        {anchor && <a id={anchor} />}
+        <Heading size={"xl"} className={"navds-layout__panel-title"} level={2}>
```

Comment:
Jeg lurer på om Heading med anchorLink burde trekkes ut som sin egen komponent, så teamene kan komponere den med panel på den måten de selv ønsker. Da har de større fleksibilitet til å gjøre ting som å velge nivå på header selv og å ha flere headinger per panel etc.

Reply:
Kanskje vi kan ha et optional anker-prop på Heading komponenten, 
og i tillegg ha et anker-prop på Panelet som videresender til Heading?

Reply:
Er ankeret noe som peker på panelet, eller peker det på overskriften?

Reply:
Akkurat for øyeblikket peker det på {anchor && <a id={anchor} />} som ligger rett over Heading, men det kan vi jo endre :) tanker?

Reply:
Har satt id-en på panelet i stedet :)